src/app/views.py

```diff
@@ -53,6 +55,44 @@ def about(request):
         'app/about.html',context_dict
         )
 
+def submitNewLicense(request):
+    """ View for submit new licenses
+    returns submit_new_license.html template
+    """
+    if request.method == 'POST':
+        form = LicenseRequestForm(request.POST, auto_id='%s')
+        if form.is_valid():
+            # process the data in form.cleaned_data as required
+            # ...
+            # redirect to a new URL:
+#            return HttpResponseRedirect('/submit_new_license/')
+    else:
+        form = LicenseRequestForm(auto_id='%s')
+    context_dict={'form': form}
+    return render(request, 
+        'app/submit_new_license.html', context_dict
+        )
+
+def generateLicenseXml(licenseOsi, licenseIdentifier, licenseName, licenseSourceUrls, licenseHeader, licenseNotes, licenseText):
```

Comment:
This function is never called. Also I think you can use the `if form.is_valid():` block in submitNewLicense function for saving the model

Reply:
Let me know if this function does something different or if you plan to do something different with this function later on.

Reply:
This function is the one that will be called to generate the xml file. I was going to call it inside the submitNewLicense function, but if you suggest I can implement it directly in submitNewLicense.

Reply:
I thought this would save the model. But if this is generating xml files, then it would be best to keep this as a separate function. But then why aren't you saving the form in submitNewLicense function?